components/quick_input.jsx

```diff
@@ -28,6 +28,7 @@ export default class QuickInput extends React.PureComponent {
 
     static defaultProps = {
         delayInputUpdate: false,
+        value: '',
```

Comment:
Can you provide any context for this change?

Reply:
Ooppss, sorry. This is not related to this PR, but it's a fix for JS error (see screenshot)
![screen shot 2018-05-15 at 10 28 20 pm](https://user-images.githubusercontent.com/5334504/40083465-cba64c2e-58c6-11e8-922e-0cac3a189950.png)